plugins/lookup/github_app_access_token.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023, Poh Wei Sheng <weisheng-p@hotmail.sg>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: github_app_access_token
+    author:
+      - Poh Wei Sheng (@weisheng-p)
+    short_description: This plugin generates Github App Access token
+    version_added: '3.1.0'
+    requirements:
+      - jwt (https://github.com/GehirnInc/python-jwt)
+    description:
+      - This generates a Github access token that can be used with a git command, if you use a Github App.
+    options:
+      key_path:
+        description:
+        - Path to your private key
```

Comment:
Generally all descriptions (except `short_description`) should consist out of paragraphs (list items) that in turn consist out of complete sentences (one or more).
```suggestion
        - Path to your private key.
```